### PR TITLE
Bring in Quietust's initial 50.01 changes

### PR DESCRIPTION
**Pull request by lethosor**
Globals aligned so far:
* init
* d_init
* world
* ui

### CODE REVIEW

File: df.buildings.xml [hunk 1]
```diff
@@ -589,7 +573,7 @@
         <enum-item/> Invalid Type
         <enum-item name="MeadHall"/>
         <enum-item name="ThroneRoom"/>
-        <enum-item name="ActivityZone"/>
+        <enum-item/> formerly ActivityZone
```
Comment (myk002) [inline df.buildings.xml:576]: is this valid syntax?

Comment (myk002) [reply df.buildings.xml:576]: I guess so -- I see it below as well for Invalid Type (under DiningHall)

Comment (lethosor) [reply df.buildings.xml:576]: yes, text nodes are treated as comments by codegen

**Comment by cvuchener** [comment]
ed3aa9d commit message says 0.50.02 but symbols.xml was not updated. Did you forget to add it?

**Comment by ab9rf** [comment]
quietust and i are working in tandem so there are also changes on my fork that should be brought forward
and as far as i know neither of us has updated symbols.xml yet for 50.02

**Comment by cvuchener** [comment]
A few values for the `unit_labors` enum have changed :

- From 15 ANIMALTRAIN to 29 BREWER are moved one further in the enum (from 16 to 30)
- 30 ALCHEMIST was removed (or moved elsewhere)
- 12 DETAIL, 13 MASON, 14 ARCHITECT are replaced by 12 Stone Cutting, 13 Stone Carving, 14 Stone Engraving, 15 Masonry

Since you are changing a lot of stuff here, I don't know if it is worth another PR for small changes. I'm leaving this comment here, if you want to pick it up.

**Comment by ab9rf** [comment]
i already have the labor changes in notes somewhere, just haven't committed them to the structures repo yet

**Comment by ab9rf** [comment]
already merged under #483

**Pull request closed without merging**